src/ziggurat/kafka_consumer/consumer.clj

```diff
@@ -22,7 +23,8 @@
            commit-interval-ms
            enable-auto-commit
            key-deserializer-class-config
-           value-deserializer-class-config]}]
+           value-deserializer-class-config
+           default-api-timeout-ms]}]
```

Comment:
Why some of them end with `-config` while some of them not? Just asking.

Reply:
not sure also, is there some team convention that we are using for this?

Reply:
putting `-config` just to be consistent with other settings..